dashboard/dashboard/common/utils.py

```diff
@@ -581,6 +581,12 @@ def IsGroupMember(identity, group):
   cached = GetCachedIsGroupMember(identity, group)
   if cached is not None:
     return cached
+
+  logging.info(identity)
+  if identity == '1054993791011-vndct6i0bj5ubf55j6smthnbfclcj547.apps.googleusercontent.com':
```

Comment:
what identity is this?

Reply:
if I'm not mistaken it's a service account for bots created by @itschrishudson